x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/perform_bulk_action_route.ts

```diff
@@ -320,6 +375,9 @@ export const performBulkActionRoute = (
               },
               abortSignal: abortController.signal,
             });
+            updated = bulkActionOutcome.results
+              .map(({ result }) => result)
+              .filter((rule): rule is RuleAlertType => rule !== null);
```

Comment:
According to https://github.com/elastic/kibana/blob/fb1eeb0945e25928a2516133055f048ff098166f/x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts#L377 (maybe that's the wrong place?), `rule` can be `null` or `undefined`. In which case `!= null` would catch both.